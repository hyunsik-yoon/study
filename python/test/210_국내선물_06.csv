거래일자,거래번호,거래종류,적요명,종목번호,종목명,수량,단가,거래금액,수수료,예수금전잔,예수금금잔,정산금액,약정금액,세금합계,대용잔고,기타수표금액,미수/변제
2023-07-11,7,매수,당일차금매수출금    ,1EQT8000,비에이치   F 202308 (  10)    ,3000,27500,15000000,79790,287243076,272163286,15079790,825000000,0,0,0,0
2023-07-12,3,입금,매수갱신차금입금    ,1EQT8000,비에이치   F 202308 (  10)    ,3000,0,12000000,0,312588286,324588286,12000000,0,0,0,0,0
2023-07-13,3,출금,매수갱신차금출금    ,1EQT8000,비에이치   F 202308 (  10)    ,3000,0,19500000,0,109823438,90323438,19500000,0,0,0,0,0
2023-07-13,12,매도,당일차금매도출금    ,1EQT8000,비에이치   F 202308 (  10)    ,500,26700,250000,0,96237978,95987978,250000,133500000,0,0,0,0
2023-07-13,13,매도,당일차금매도입금    ,1EQT8000,비에이치   F 202308 (  10)    ,1024,26750,0,0,95987978,95987978,0,273920000,0,0,0,0
2023-07-13,14,매도,당일차금매도입금    ,1EQT8000,비에이치   F 202308 (  10)    ,2,26800,1000,0,95987978,95988978,1000,536000,0,0,0,0
2023-07-13,15,매도,당일차금매도입금    ,1EQT8000,비에이치   F 202308 (  10)    ,1,26850,1000,0,95988978,95989978,1000,268500,0,0,0,0
2023-07-13,16,매도,당일차금매도입금    ,1EQT8000,비에이치   F 202308 (  10)    ,423,26900,634500,50480,95989978,96573998,584020,113787000,0,0,0,0
2023-07-14,2,입금,매수갱신차금입금    ,1EQT8000,비에이치   F 202308 (  10)    ,1050,0,525000,0,98323998,98848998,525000,0,0,0,0,0
2023-07-17,2,입금,매수갱신차금입금    ,1EQT8000,비에이치   F 202308 (  10)    ,1050,0,12075000,0,100098998,112173998,12075000,0,0,0,0,0
2023-07-18,1,출금,매수갱신차금출금    ,1EQT8000,비에이치   F 202308 (  10)    ,1050,0,5250000,0,112173998,106923998,5250000,0,0,0,0,0
2023-07-19,1,출금,매수갱신차금출금    ,1EQT8000,비에이치   F 202308 (  10)    ,1050,0,5250000,0,102923998,97673998,5250000,0,0,0,0,0
2023-07-20,2,출금,매수갱신차금출금    ,1EQT8000,비에이치   F 202308 (  10)    ,1050,0,525000,0,97673998,97148998,525000,0,0,0,0,0
2023-07-21,2,출금,매수갱신차금출금    ,1EQT8000,비에이치   F 202308 (  10)    ,1050,0,2625000,0,99648998,97023998,2625000,0,0,0,0,0
2023-07-24,1,출금,매수갱신차금출금    ,1EQT8000,비에이치   F 202308 (  10)    ,1050,0,3150000,0,97023998,93873998,3150000,0,0,0,0,0
2023-07-25,1,입금,매수갱신차금입금    ,1EQT8000,비에이치   F 202308 (  10)    ,1050,0,1575000,0,92373998,93948998,1575000,0,0,0,0,0
2023-07-26,1,출금,매수갱신차금출금    ,1EQT8000,비에이치   F 202308 (  10)    ,1050,0,7350000,0,90198998,82848998,7350000,0,0,0,0,0
2023-07-27,1,출금,매수갱신차금출금    ,1EQT8000,비에이치   F 202308 (  10)    ,1050,0,8925000,0,80848998,71923998,8925000,0,0,0,0,0
2023-07-28,2,입금,매수갱신차금입금    ,1EQT8000,비에이치   F 202308 (  10)    ,1050,0,4200000,0,69173998,73373998,4200000,0,0,0,0,0
2023-07-31,1,출금,매수갱신차금출금    ,1EQT8000,비에이치   F 202308 (  10)    ,1050,0,10500000,0,73373998,62873998,10500000,0,0,0,0,0
2023-08-01,2,출금,매수갱신차금출금    ,1EQT8000,비에이치   F 202308 (  10)    ,1050,0,6300000,0,64373998,58073998,6300000,0,0,0,0,0
2023-08-01,4,매도,당일차금매도입금    ,1EQT8000,비에이치   F 202308 (  10)    ,50,23800,25000,1150,57972968,57996818,23850,11900000,0,0,0,0
2023-08-02,1,입금,매수갱신차금입금    ,1EQT8000,비에이치   F 202308 (  10)    ,1000,0,9500000,0,57996818,67496818,9500000,0,0,0,0,0
2023-08-03,1,입금,매수갱신차금입금    ,1EQT8000,비에이치   F 202308 (  10)    ,1000,0,2500000,0,66596818,69096818,2500000,0,0,0,0,0
2023-08-04,2,출금,매수갱신차금출금    ,1EQT8000,비에이치   F 202308 (  10)    ,1000,0,7500000,0,65046818,57546818,7500000,0,0,0,0,0
2023-08-07,1,출금,매수갱신차금출금    ,1EQT8000,비에이치   F 202308 (  10)    ,1000,0,7000000,0,57546818,50546818,7000000,0,0,0,0,0
2023-08-07,5,매도,당일차금매도출금    ,1EQT8000,비에이치   F 202308 (  10)    ,10,23300,20000,220,49367908,49347688,20220,2330000,0,0,0,0
2023-08-08,2,출금,매수갱신차금출금    ,1EQT8000,비에이치   F 202308 (  10)    ,990,0,1980000,0,51597688,49617688,1980000,0,0,0,0,0
2023-08-09,1,출금,매수갱신차금출금    ,1EQT8000,비에이치   F 202308 (  10)    ,990,0,4455000,0,49617688,45162688,4455000,0,0,0,0,0
2023-08-10,2,입금,매수갱신차금입금    ,1EQT8000,비에이치   F 202308 (  10)    ,990,0,495000,0,47149399,47644399,495000,0,0,0,0,0
2023-08-10,5,매도,당일차금매도입금    ,1EQT8000,비에이치   F 202308 (  10)    ,1090,23000,1090000,24240,41883289,42949049,1065760,250700000,0,0,0,0
2023-08-10,8,매수,당일차금매수출금    ,1EQT8000,비에이치   F 202308 (  10)    ,100,23050,150000,2220,48936919,48784699,152220,23050000,0,0,0,0
