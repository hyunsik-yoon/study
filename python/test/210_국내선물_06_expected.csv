종목명,진입일자,진입단가,청산일자,청산단가,거래수량,총수수료,총세금합계,순수익
비에이치   F 202308 (  10), 07/11/2023, 27500, 07/13/2023, 26700, 500, 13298, 0, -4000000
비에이치   F 202308 (  10), 07/11/2023, 27500, 07/13/2023, 26750, 1024, 27235, 0, -7680000
비에이치   F 202308 (  10), 07/11/2023, 27500, 07/13/2023, 26800, 2, 53, 0, -14000
비에이치   F 202308 (  10), 07/11/2023, 27500, 07/13/2023, 26850, 1, 27, 0, -6500
비에이치   F 202308 (  10), 07/11/2023, 27500, 07/13/2023, 26900, 423, 61730, 0, -2538000
비에이치   F 202308 (  10), 07/11/2023, 27500, 08/01/2023, 23800, 50, 2480, 0, -1850000
비에이치   F 202308 (  10), 07/11/2023, 27500, 08/07/2023, 23300, 10, 486, 0, -420000
비에이치   F 202308 (  10), 07/11/2023, 27500, 08/10/2023, 23000, 990, 48347, 0, -44550000
비에이치   F 202308 (  10), 08/10/2023, 23000, 08/10/2023, 23050, 100, 4444, 0, -50000
